### PR TITLE
Fix stale info about spread minDomains

### PR DESCRIPTION
**Pull request by tamilselvan1102**
fixes https://github.com/kubernetes/website/issues/42670

### CODE REVIEW

**Comment by tamilselvan1102** [comment]
preview page: https://deploy-preview-42758--kubernetes-io-main-staging.netlify.app/docs/concepts/scheduling-eviction/topology-spread-constraints/#spread-constraint-definition

**Comment by sftim** [comment]
/retitle Fix stale info about spread minDomains

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 1]
```diff
@@ -97,8 +97,11 @@ your cluster. Those fields are:
   nodes match the node selector.
 
   {{< note >}}
-  The `minDomains` field is a beta field and disabled by default in 1.25. You can enable it by enabling the
-  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+  The `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enables `minDomains` in pod 
+  topology spread. From v1.28 release, `MinDomainsInPodTopologySpread` gate 
+  is enabled by default and older clusters it might have explicitly
```
Comment (tengqm) [inline content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:102]: This sentence doesn't flow well, pls double check.

File: content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md [hunk 2]
```diff
@@ -97,8 +97,10 @@ your cluster. Those fields are:
   nodes match the node selector.
 
   {{< note >}}
-  The `minDomains` field is a beta field and disabled by default in 1.25. You can enable it by enabling the
-  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
+  The `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) enables `minDomains` in pod 
+  topology spread. From v1.28 release, `MinDomainsInPodTopologySpread` gate
```
Comment (tengqm) [inline content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md:101]:
```suggestion
  The `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
  enables `minDomains` for pod topology spread. Starting from v1.28,
  the `MinDomainsInPodTopologySpread` gate 
```

**Review by sftim — COMMENTED** [review]
Thanks

/approve

**Pull request merged**